charts/auto-scaling-runner-set/values.yaml

```diff
@@ -13,6 +13,10 @@ githubConfigSecret:
 
   ### GitHub PAT Configuration
   github_token: ""
+## If you have a pre-define kubernetes secret in the same namespace the auto-scaling-runner-set is going to deploy,
+## you can also reference it via `githubConfigSecret: predefinedSecret`.
+## You need to make sure your predefined secret has all required secret data set properly, ex, github_token/github_app_id/github_app_installation_id/github_app_private_key
```

Comment:
nit: maybe we add an example for how to create the secret with the appropriate structure in our docs and reference it here?